docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -30,6 +30,7 @@ EdgeX snaps are packaged with default service configuration files. In certain ca
 #### Config files
 The default configuration files are typically placed at `/var/snap/<snap>/current/config`. Upon startup, the server configurations files are uploaded to the registry by default. Once the service starts without errors, the local configurations become obsolete and will no longer be read. Any modifications after the initial startup will not be applied. 
 
+For device services, the configurations of Device and Device Profile are submitted to [Core Metadata](../../microservices/core/metadata/Ch-Metadata) upon startup. Refer to the documentation of [Device Services](../../microservices/device/Ch-DeviceServices/) for details.
```

Comment:
how about adding a device example here? since there are several paragraphs in `Configuration` section, maybe an image may help the user understand.
```
/var/snap/edgex-device-<name>/current/config/
└── device-<name>
    └── res
        ├── configuration.toml
        ├── devices
        │   ├── device.1.yml
        │   └── device.2.toml
        └── profiles
            ├── device.profile.yml
            ├── device.profile.2.yml
            └── device.profile.3.yml
```

Reply:
Examples are already provided in snap sections. The path where the files are is given above. I think adding more details will be confusing at this port because app, devices, and platform have different structures.